app/controllers/api/red_hat_migration_analytics_controller.rb

```diff
@@ -1,6 +1,7 @@
 module Api
   class RedHatMigrationAnalyticsController < BaseController
     def index
+      check_feature_enabled
```

Comment:
If we're disabling the route I wonder if this is just duplicating that check

Reply:
The check I added in routes.rb just disables the `/migration_analytics` route which renders the UI, I couldn't find a good way to disable the `/api/red_hat_migration_analytics` route other than this